docs/tutorials/go/audiobook/index.md

```diff
@@ -0,0 +1,1026 @@
+---
+id: audiobook-tutorial
+keywords: [Go, temporal, sdk, tutorial, entity workflow, audiobook, text to speech, OpenAI]
+tags: [Go, SDK]
+last_update:
+  date: 2024-08-07
+title: Create audiobooks from text with OpenAI and Go
+description: Learn to build your own audiobooks from text using OpenAI Web APIs and Temporal. Step-by-step guide for hassle-free MP3 creation with robust failure mitigation.
+sidebar_label: Create audiobooks from text with OpenAI and Go
+image: /img/temporal-logo-twitter-card.png
+---
+
+![Temporal Go SDK](/img/sdk_banners/banner_go.png)
+
+## Introduction
+
+For many people, audiobooks are a staple of daily life.
+Spoken content helps people pass time, learn new things, and stay entertained without tying themselves to a screen or a book.
+Imagine turning your daily commute, workout sessions, or household chores into opportunities to catch up on _all_ your reading -- not just books.
+You can grab text from emails, from the web, or even that report you need to "read" for work.
+With the rise of advanced text-to-speech (TTS) technology, it's not just possible, it's quick to build in Go with simple APIs.
+
+For years, text-to-speech solutions have supported the assistive community by transforming words on the page into accessible engagement.
+However, until recently, robotic voices and unnatural rhythms made TTS unappealing for casual listeners.
+If you didn't need TTS, you probably wouldn't have used TTS.
+Now, companies like [OpenAI](https://openai.com/api/) have revolutionized speech synthesis.
```

Comment:
I assume this language already made it into the tutorial this was ported from so it's not a strong objection, but this is, imo, pitching OpenAI a little too hard

Reply:
Brian said "let OpenAI be the star" (more or less).